HW0/P6/P6.txt

```diff
@@ -1 +1,5 @@
 # Your answers here
```

Comment:
The graph for P6 looks very noisy and doesn't seem to indicate much of a speed-up when moving to 4 processes.